content/en/docs/components/pipelines/v2/cli.md

```diff
@@ -4,119 +4,156 @@ description = "Interact with KFP via the CLI"
 
-You can check that the KFP CLI is installed in your environment by running:
+### Check availability of KFP CLI
+
+To check whether KFP CLI is installed in your environment, run the following command:
 
 ```shell
 kfp --version
 ```
 
-All commands assume the general structure:
+### General syntax
+
+All commands in the KFP CLI use the following general syntax:
 
 ```shell
 kfp [OPTIONS] COMMAND [ARGS]...
 ```
 
-For example, to list all runs for a specific endpoint:
+For example, to list all runs for a specific endpoint, run the following command:
 
 ```shell
 kfp --endpoint http://my_kfp_endpoint.com run list
 ```
 
-You can get help for a particular command directly on the command line using `--help`:
+### Get help for a command
+
+To get help for a specific command, use the argument `--help` directly in the command line. For example, to view guidance about the `kfp run` command, type the following command:
 
 ```shell
 kfp run --help
 ```
+## Main functons of the KFP CLI
+
+You can use the KFP CLI to do the following:
 
-The KFP CLI serves three main functions:
-1. Interacting with KFP resources
-2. Compiling pipelines
-3. Building Python-based containerized components
+* [Interact with KFP resources](#interact_with_kfp_resources)
 
-## Interact with KFP resources
-The majority of the KFP CLI commands involve creating, reading, updating, or deleting KFP resources via the KFP backend. All of these commands take a common structure:
+* [Compile pipelines](#compile_pipelines)
+
+* [Build containerized Python components](#build_containerized_python_components)
+
+### Interact with KFP resources
+
+The majority of the KFP CLI commands let you create, read, update, or delete KFP resources from the KFP backend. All of these commands use the following general syntax:
 
 ```shell
 kfp <resource_name> <action>
 ```
 
-`<resource_name>` can be one of the following:
+The `<resource_name>` argument can be one of the following:
 * `run`
 * `recurring-run`
 * `pipeline`
 * `experiment`
 
-All resources support the following actions:
+For all values of the `<resource_name>` argument, the `<action>` argument can be one of the following:
 * `create`
 * `list`
 * `get`
 * `delete`
 
-Some resources have resource-specific actions, including:
-* For `run`:
-  * `archive`
-  * `unarchive`
-
-* For `recurring-run`:
-  * `disable`
-  * `enable`
-
-* For `experiment`
-  * `archive`
-  * `unarchive`
+Some resource names have additional resource-specific actions. The following table lists a few examples of resource-specific actions:
+
+| Resource name | Additional resource-specific actions
+|---------------|--------
+| `run` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `recurring-run` | <ul><li>`disable`</li><li>`enable`</li></ul>
+| `experiment` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `pipeline` |  | <ul><li>`create-version`</li><li>`list-versions`</li><li>`get-versions`</li><li>`delete-versions`</li></ul>
```

Comment:
The bottom right column of this table isn't rendering on the preview. Can you take a look?

Reply:
Fixed it.